### PR TITLE
feat : add typograpghy,lists and card

### PR DESCRIPTION
**Pull request by shruthipatil11**
Typography, lists and card components

### CODE REVIEW

File: src/app/components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  text-align: center;
+  flex-wrap: wrap;
+  min-height: 7rem;
+  row-gap: 10px;
+  font-size: 1.2rem;
+}
+
+.card__image {
+  max-width: 18rem;
+  /* min-width: 20rem; */
```
Comment (ashishkatiyar2404) [inline src/app/components/cards/cards.css]: you can remove comments

File: src/app/components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  min-height: inherit;
+}
+
+.card--text-overlay {
+  position: relative;
+}
+
+.card--text-overlay .text-overlay {
+  position: absolute;
+  top: 0%;
```
Comment (ashishkatiyar2404) [inline src/app/components/cards/cards.css:94]: when you use position try to give both property (top with left/right or bottom with left/right)

File: src/app/components/typography/typography.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
+#typography .components__parts .h1 {
```
Comment (ashishkatiyar2404) [inline src/app/components/typography/typography.css]: pls try to avoid ID variable for styling

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested some change...
your code looks good to merge💯

File: src/app/components/lists/lists.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    border-color: var(--INFO-BORDER);
+}
+
+.components__stacked-list li:nth-child(3) {
+    color: var(--WARNING-MESSAGE);
+    background-color: var(--WARNING-BACKGROUND);
+    border-color: var(--WARNING-BORDER);
+}
+
+
```
Comment (muskan-gupta-0803) [inline src/app/components/lists/lists.css]: remove unnecessary blank spaces in code

**Review by muskan-gupta-0803 — APPROVED** [review]
Just make minor changes as suggested.
Otherwise it looks good to merge.

File: src/app/components/cards/cards.css (hunk 1 above)
Comment (shruthipatil11) [reply src/app/components/cards/cards.css]: done

File: src/app/components/cards/cards.css (hunk 2 above)
Comment (shruthipatil11) [reply src/app/components/cards/cards.css:94]: done. Thanks for let me know.

File: src/app/components/typography/typography.css (hunk 3 above)
Comment (shruthipatil11) [reply src/app/components/typography/typography.css]: removed unwanted ids as suggested , used id at one place now as that is required .Using id when absolutely required is not an issue is my understanding .

File: src/app/components/lists/lists.css (hunk 4 above)
Comment (shruthipatil11) [reply src/app/components/lists/lists.css]: done

**Pull request merged**